### ISSUE TITLE
Starting Download in installer crashes network connection

### ISSUE THREAD

**Issue by screeb7165** [SkyrimTogether/issues-launcher#18]
## Description

Downloaded and installed Harbor. Set game folder location. Hit install button for Skyrim Together. First time, internet connection dropped immediately. Second time, got about 5% through install, network connection crashed again. Other devices on network unaffected. Could ping and connect to router, could not reach DNS or other locations beyond local network.

## Steps to reproduce

Run installation for Skyrim Together. Happens almost right away.

## Reproduction rate

Happened twice. Did not want to test further, as other people are on the network and I had to restart the network both times to restore connection.

## Expected result

Ability to install mod without crashing network.

## Your environment

* Launcher version: 1.0.5

* Game edition (choose on which edition do you have problems):
    * The Elder Scrolls V: Skyrim Special Edition

* Operating system (choose one):
    * Windows 10 x64

## Final checks for the reporter

- [X] I have browsed through the other issues and confirmed that this issue has not been reported already
- [X] I made sure that I have applied the template for my bug report

**Comment by Pokang** [comment]
So let me get this right, when you tried downloading the mod through our launcher it crashes your whole network?

**Comment by screeb7165** [comment]
No, for whatever reason, the device I was downloading the mod on lost internet access. All other devices were unaffected, and I could still communicate with other devices on the network, but short of restarting the router, I couldn't get my computer to talk to anything on the other side of the router.

**Comment by WopsS** [comment]
Thank you for confirming, we will have a look into it.
